velox/functions/sparksql/Register.cpp

```diff
@@ -402,6 +402,8 @@ void registerFunctions(const std::string& prefix) {
       {prefix + "unix_millis"});
   registerUnaryIntegralWithTReturn<MillisToTimestampFunction, Timestamp>(
       {prefix + "timestamp_millis"});
+  registerUnaryFloatingWithTReturn<SecondsToTimestampFunction, Timestamp>(
+        {prefix + "timestamp_seconds"});
```

Comment:
Spark supports more types than double. Do we need to support them?

Reply:
Yeah, I suppose maybe align with Spark is a better idea. Will support more types in the next iteration.